session/src/main/scala/akka/cassandra/session/scaladsl/CassandraSession.scala

```diff
@@ -72,7 +82,10 @@ final class CassandraSession(
    */
   def underlying(): Future[CqlSession] = _underlyingSession
 
-  def close(): Unit = _underlyingSession.foreach(_.close())
+  def close(): Future[Done] = {
```

Comment:
good

Reply:
Java API naming? difficult, maybe pick one of the synonyms `shutdown` or `terminate`